### PR TITLE
feat: add wallet address

### LINKED ISSUE #3569: Add wallet address to the `/wallet` endpoint
Currently the wallet endpoint returns a field called `contractAddress` and that is the address of the chequebook contract. However it is missing the actual wallet address, which is the same as the ethereum address of the node. It would be good to have it as well.

### PR DESCRIPTION
**Pull request by umerm-work**
### Checklist

- [x] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [x] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
Closes #3569 

### Screenshots (if appropriate):

- - -
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/3571)

### CODE REVIEW

File: pkg/api/wallet.go [hunk 1]
```diff
@@ -13,16 +13,17 @@ import (
...
 
 type walletResponse struct {
-	BZZ             *bigint.BigInt `json:"bzz"`             // the BZZ balance of the wallet associated with the eth address of the node
-	XDai            *bigint.BigInt `json:"xDai"`            // the xDai balance of the wallet associated with the eth address of the node
-	ChainID         int64          `json:"chainID"`         // the id of the block chain
-	ContractAddress common.Address `json:"contractAddress"` // the address of the chequebook contract
+	BZZ             *bigint.BigInt `json:"bzzBalance"`         // the BZZ balance of the wallet associated with the eth address of the node
+	NativeToken     *bigint.BigInt `json:"nativeTokenBalance"` // the xDai balance of the wallet associated with the eth address of the node
+	ChainID         int64          `json:"chainID"`            // the id of the blockchain
+	ContractAddress common.Address `json:"contractAddress"`    // the address of the chequebook contract
```
Comment (istae) [inline pkg/api/wallet.go:19]: rename to `ChequebookContractAddress`

**Review by istae — CHANGES_REQUESTED** [review]

File: pkg/api/wallet.go [hunk 2]
```diff
@@ -13,16 +13,17 @@ import (
 )
 
 type walletResponse struct {
-	BZZ             *bigint.BigInt `json:"bzz"`             // the BZZ balance of the wallet associated with the eth address of the node
-	XDai            *bigint.BigInt `json:"xDai"`            // the xDai balance of the wallet associated with the eth address of the node
-	ChainID         int64          `json:"chainID"`         // the id of the block chain
-	ContractAddress common.Address `json:"contractAddress"` // the address of the chequebook contract
+	BZZ                       *bigint.BigInt `json:"bzzBalance"`                // the BZZ balance of the wallet associated with the eth address of the node
+	NativeToken               *bigint.BigInt `json:"nativeTokenBalance"`        // the xDai balance of the wallet associated with the eth address of the node
```
Comment (istae) [inline pkg/api/wallet.go:17]: change comment

**Review by istae — CHANGES_REQUESTED** [review]

**Review by istae — APPROVED** [review]

**Comment by mrekucci** [comment]
The changes need to be reflected in the OpenAPI spec.

**Review by agazso — APPROVED** [review]
Looks good!

**Comment by umerm-work** [comment]
> The changes need to be reflected in the OpenAPI spec.

Thanks for pointing. Added.

**Pull request merged**